source/slang/slang-ir-specialize.cpp

```diff
@@ -605,6 +603,15 @@ struct SpecializationContext
 
         case kIROp_CountOf:
             return maybeSpecializeCountOf(inst);
+
+        case kIROp_Func:
+
+            if (tryExpandParameterPack(as<IRFunc>(inst)))
```

Comment:
Yong's note: in this new specialization logic, if we see a function that takes a TypePack parameter, we are just going to expand that type pack out of existence. For example, if we have `int f(TypePack<int, float> v)`, we are going to turn it into `int f(int v0, float v1)`. This will make the final emitted code much cleaner, and is the easiest way to make interface method with variadic parameters work so we don't have to worry about packing ordianry arguments into packs.